test/e2e/filter_test.go

```diff
@@ -62,7 +62,7 @@ data:
 			RunOpts{IntoNs: true, StdinReader: strings.NewReader(yaml1)})
 		expectedOutput1 := `
 Namespace  Name           Kind       Age  Op      Op st.  Wait to    Rs  Ri  
-kapp-test  redis-config   ConfigMap  -    create  -       reconcile  -   -  
+` + env.Namespace + `  redis-config   ConfigMap  -    create  -       reconcile  -   -  
```

Comment:
Maybe use `fmt` or `strings` to format or replace namespace?

Also, outputs like these would need some special handling as the indentation would change based on the env.Namespace value. We could create separate PR to keep the changes minimal for now, but feel free to make the changes in the same PR itself.

Reply:
`strings` would be a good alternative. Though as you mentioned this might fail due to indentation. Currently, I don't have a solution to handle varying indentation.

Reply:
> Currently, I don't have a solution to handle varying indentation.

That's okay, we can have a separate PR to handle this. You can either leave it as it is for now (i.e. with hardcoded "kapp-test")